frontend/packages/knative-plugin/src/components/overview/KnativeOverview.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ResourceSummary } from '@console/internal/components/utils';
+import { OverviewItem, PodRing } from '@console/shared';
+import { RevisionModel } from '../../models';
+
+export type KnativeOverviewProps = {
+  item?: OverviewItem;
+};
+
+export const KnativeOverview: React.FC<KnativeOverviewProps> = ({ item }) => {
+  const { obj, current } = item;
+  return (
+    <div className="overview__sidebar-pane-body resource-overview__body">
+      {obj.kind === RevisionModel.kind && (
```

Comment:
is there a possibility for `obj.kind` to be present and not equal to `RevisionModel.kind`